### ISSUE TITLE
0.33: missing assets

### ISSUE THREAD

**Issue by mathstuf** [obi1kenobi/cargo-semver-checks#836]
### Steps to reproduce the bug with the above code

Only Github-generated source archive assets are available: https://github.com/obi1kenobi/cargo-semver-checks/releases/tag/v0.33.0

### Actual Behaviour

No assets available.

### Expected Behaviour

Assets available like 0.32.

### Generated System Information

N/A

**Comment by mathstuf** [comment]
Ah, this action failed: https://github.com/obi1kenobi/cargo-semver-checks/actions/runs/10084016103/job/27882323989

**Comment by mathstuf** [comment]
Because the workflows were updated during the release, Github refuses a push updating them from a workflow:

```
To https://github.com/obi1kenobi/cargo-semver-checks
 ! [remote rejected] v0.33.0 -> v0.33.0 (refusing to allow a GitHub App to create or update workflow `.github/workflows/ci.yml` without `workflows` permission)
error: failed to push some refs to 'https://github.com/obi1kenobi/cargo-semver-checks'
```

**Comment by obi1kenobi** [comment]
Fix part 1: #837

I'll use that to build the missing 0.33 assets. Then I'll amend the tagging portion of the workflow so it doesn't fail again in the future.

**Comment by obi1kenobi** [comment]
Fix part 2: #838

Hopefully that prevents the problem from coming back.

Apologies for breaking your workflow, and thanks for your patience!

**Comment by mathstuf** [comment]
Thanks!
